### ISSUE TITLE
Reads are disabled, although my Micropub server advertises support

### ISSUE THREAD

**Issue by jamietanna** [swentel/indigenous-android#410]
**Describe the bug**

When enabling the setting `Hide post types`, the ability to post a read is disabled.

**To Reproduce**

See additional context for `q=config` responses.

**Expected behavior**

Publishing reads would be possible.

**Screenshots**
N/A

**Smartphone (please complete the following information):**
 - Device: Pixel
 - Android version: 3A

**Additional context**

I have tried with the following in my `q=config`:

```json
{
  "post-types": [
    {
      "h": "entry",
      "name": "Reading",
      "properties": [
        "published",
        "read-of",
        "read-status"
      ],
      "required-properties": [
        "published",
        "read-of",
        "read-status"
      ],
      "type": "read"
    },
  ]
}
```

And

```json
{
  "post-types": [
    {
      "h": "entry",
      "name": "Reading",
      "properties": [
        "published",
        "read-of",
        "read-status"
      ],
      "required-properties": [
        "published",
        "read-of",
        "read-status"
      ],
      "type": "reads"
    },
  ]
}
```

But neither work :thinking: Happy to be told there's a different `type` we're using!

**Comment by swentel** [comment]
Type should be "read" so the first one should be fine normally. I'm assuming you refreshed the configuration as well in Indigenous?

This is about the main menu right? Wanted to check that, because there's also a 'read' configuration option for the reader (as there's such a close difference to bookmark too, I made those responses for every item optional).

**Comment by jamietanna** [comment]
Yep, I've just retried it, making sure to refresh config, and it still doesn't show up.

Yep, this is from the main menu's `Write` option

**Comment by swentel** [comment]
Just tried it with my micropub server adding 'read', and it showed up in the Write section (after a restart of indigenous too, that' something I need to fix). When I disabled it again on my server, refreshed config, it was gone again too.

So not clue why this is broken. Apart from maybe the fact that you already run Android 11? If so, I'll have to test with an emulator as I'm still on 9.
